### PR TITLE
Update ghc v8.10.1

### PR DESCRIPTION
**Pull request by rewinfrey**
This is a companion to https://github.com/tclem/proto-lens-jsonpb/pull/6 and updates `twirp-haskell` to use GHC v8.10.1.

### CODE REVIEW

File: cabal.project [hunk 1]
```diff
@@ -1,3 +1,8 @@
 packages: . example
 
 jobs: $ncpus
+
+source-repository-package
+  type: git
+  location: https://github.com/tclem/proto-lens-jsonpb
+  tag: ce102da9fd7f7b405df414ee571aa12ee60c85bd
```
Comment (tclem) [inline cabal.project]: nit: newlines at end of file here and in the cabal.project below please! 🙇‍♂️

**Review by tclem — COMMENTED** [review]
I did release a new version of https://hackage.haskell.org/package/proto-lens-jsonpb-0.2.0.1 this week and it's easy to do another. Do we want to depend on git repos or on hackage?

**Comment by rewinfrey** [comment]
@tclem, imo it's nicer to depend on hackage versions b/c it keeps us honest and ensures new versions are published. I can wait until the `proto-lens-jsonpb` patch lands and is published to hackage, and then update this patch to remove the `git` references.

**Comment by rewinfrey** [comment]
I've removed the git reference for `proto-lens-jsonpb` from `cabal.project`, and have uploaded the updated package to hackage: https://hackage.haskell.org/package/twirp-0.2.0.1

**Comment by rewinfrey** [comment]
@tclem I also added the `build-and-upload` script to this repo for convenience. It's a copy of the same script used in https://github.com/tclem/proto-lens-jsonpb/blob/master/script/build-and-upload

**Pull request merged**